packages/patternfly-3/patternfly-react-extensions/src/components/DiffView/DiffView.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { parseDiff, Diff, markCharacterEdits } from 'react-diff-view';
+import { formatLines, diffLines } from 'unidiff';
+
+const getDiff = (oldText, newText) => {
+  const diffText = formatLines(diffLines(oldText, newText), { context: 3 });
+  // these two lines are faked to mock git diff output
+  const header = ['diff --git a/a b/b', 'index 0000000..1111111 100644'];
```

Comment:
Values will be inaccurate right? This could be a default value for a prop allowing apps to decide if they want it or not or to customize the header.

Reply:
I don't think that apps who input old, new text should worry about this.
This function creates a git diff output out of those texts, the DiffView needs it to render.

Reply:
I guess I don't understand the comment

Reply:
nevermind, ill make it a prop :)